README.adoc

```diff
@@ -1527,6 +1527,34 @@ else
 end
 ----
 
+=== `unless` with Multiple Conditions
+
+Do not use `unless` with multiple conditions.
+
+[source,ruby]
+----
+# bad
+unless sweet?(fruit) && !fruit.green?
```

Comment:
For me, the problem isn't so much the multiple conditions, if it's the mixing of `unless` with the inversion. I'd have no problem with `unless sweet?(fruit) && fruit.green?`.

Let's try get some wider opinions – but I think this could be a tricky area to reach a consensus on.

Reply:
Yeah, I agree we should have a separate listing for an "OKish" use-case, involving more straightforward cases.

Reply:
I'm perfectly fine with narrowing this guideline down to:

> Do not use `unless` with multiple conditions that involve negation.

And adding example `unless sweet?(fruit) && fruit.green?` to OK'ish section.

Is this sufficient to convience you guys?

Reply:
Well, I think things become dicey the moment you write something like `unless x && y || z`. My point was that a single `||` or `&&` without negation is ok for some people, but generally everything else is a bad idea in terms of readability.